README.md

```diff
@@ -55,6 +55,109 @@ fn main() {
 }
 ```
 
+# Extensions and Profiles
```

Comment:
I am wondering whether using something more realistic than "my_profile" and "ext1" is better in code that has the quality of becoming "exemplary" ;-)